src/components/common/Pagenation/ui-pagenation-wrap/PagenationWrap.jsx

```diff
@@ -0,0 +1,20 @@
+export default function PagenationWrap({ children, start, end, handleOnClick, handleOnEnter }) {
+  const onClick = (e) => {
+    handleOnClick(e.target.value);
+  };
+  const onEnter = (e) => {
+    handleOnEnter(e.target.value);
+  };
+
+  return (
+    <>
+      <button onClick={onClick} onMouseEnter={onEnter} value={start}>
```

Comment:
```suggestion
      <button onClick={(e) => handleOnClick(e.target.value)} onMouseEnter={(e) => handleOnEnter(e.target.value)} value={start}>
```
함수를 따로 생성하지 않고 이렇게 inline으로 적용해도 좋을 것 같습니다.

Reply:
수정했습니다.